Assets/Scripts/Subtitles.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class Subtitles : MonoBehaviour
```

Comment:
Change name to Subtitle? xd

Reply:
I mean... it configures the "subtitles" right? 😅

Reply:
"Subtitles" mean the whole set of them (here e.g. they are in arrays), while "subtitle" is one piece of subtitle text